packages/react-router-dom/docs/guides/quick-start.md

```diff
@@ -4,11 +4,10 @@ You'll need a React web app to add `react-router`.
 
 If you need to create one, the easiest way to get started is with a popular tool called [Create React App][crapp].
 
-First install `create-react-app`, if you don't already have it, and then make a new project with it.
+The easiest way to run `create-react-app` is with `npx`.
```

Comment:
I generally try to avoid using "easy" (and [similar words](https://twitter.com/JessTelford/status/992756386160234497)). I would amend this and the previous line to something like (feel free to play around with the wording):

> If you need to create one, the popular tool [Create React App][crapp] can get you started quickly. Calling `npx create-react-app` will create an application and install its dependencies for you.

Reply:
Thanks @pshrmn, good spotting - i'll update it.